earthfile2llb/converter.go

```diff
@@ -1140,7 +1140,10 @@ func (c *Converter) Expose(ctx context.Context, ports []string) error {
 	}
 	c.nonSaveCommand()
 	for _, port := range ports {
-		c.mts.Final.MainImage.Config.ExposedPorts[port] = struct{}{}
+		if strings.Contains(port, ":") {
+			port = strings.SplitN(port, ":", 2)[1]
+		}
+		c.mts.Final.MainImage.Config.ExposedPorts[port + "/tcp"] = struct{}{}
```

Comment:
assuming it's `tcp` here seems incorrect; is it possible for the user to expose a port for `udp`?

Reply:
fixed